bbman.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="bbman.js"></script>
```

Comment:
Please remove two lines [5,6] above this comment. Those aren't in standard emmet template - MS added them god knows why.